service/devdoc/registry_requirements.md

```diff
@@ -61,6 +61,26 @@ The `fromSharedAccessSignature` static method returns a new instance of the `Reg
 
 ## CRUD operation for the device registry
 
+### normalize authentication
+With 2017-06-30 api release crud device information MUST contain authentication specification
+
+**SRS_NODE_IOTHUB_REGISTRY_06_028: [** A device information with no authentication will be normalize with the following authentication:
```

Comment:
Fiexed.

---
In reply to: [127059528](https://github.com/Azure/azure-iot-sdk-node/pull/61#discussion_r127059528) [](ancestors = 127059528)